content/machine-controller/cloud-providers/vsphere/_index.en.md

```diff
@@ -0,0 +1,275 @@
++++
+title = "VMWare vSphere"
+date = 2024-05-31T07:00:00+02:00
++++
+
+## Supported versions
+
+* 6.5
+* 6.7
+
+## Template VMs preparation
```

Comment:
We should consider dropping this and linking to the document that we have in the KubeOne docs

Reply:
I feel extremely uneasy about relying on a 3rd party project, that builds on the machine-controller, for documenting the machine-controller. Why should KubeOne document how the MC works?

Reply:
Can / Should we move the K1 docs for image management to MC?

Reply:
I think that would make sense, to move the documents to the MC docs